Makefile

```diff
@@ -1,17 +1,38 @@
 MAKEFLAGS += --silent
 SHELL = /usr/bin/env bash
 
+_IMAGE_NAME = duolingo/pre-commit-hooks
+
 # Bumps this project's version number. Example:
+#
 #   $ make release V=1.0.3
-release:
+#
+# After running this, you should push master and tags to GitHub and create a
+# corresponding GitHub release.
+release: test
+	# Validate
 	[[ -n "${V}" ]]
 	[[ -z "$$(git status --porcelain)" ]]
+
+	# Update source files and commit
 	git grep --cached -z -l '' | xargs -0 sed -E -i \
-		"s@( rev: | entry: duolingo/pre-commit-hooks:)$$(git tag | tail -1)@\1${V}@g"
+		"s@( rev: | entry: $(_IMAGE_NAME):)$$(git tag | tail -1)@\1${V}@g"
 	git add -A
 	git commit -m "Release ${V}" -n
+
+	# Tag
 	git tag "${V}"
 
+# Pushes to Docker Hub. Should be run when creating a GitHub release. We don't
+# use Docker Hub's autobuild feature anymore because it only supports amd64 :/
+# https://github.com/docker/roadmap/issues/109
+push: test
+	docker buildx build --platform linux/amd64,linux/arm64 \
+		-t "$(_IMAGE_NAME):$(git tag | tail -1)" \
+		-t "$(_IMAGE_NAME):latest" \
+		.
+	docker push "$(_IMAGE_NAME)"
```

Comment:
I think you actually need to include the `--push` in the command itself. If you build an image using `buildx`, that image isn't available in the regular docker